src/main/java/uk/gov/digital/ho/proving/income/audit/AuditClient.java

```diff
@@ -51,15 +53,19 @@
                 RestTemplate restTemplate,
                 RequestData requestData,
                 @Value("${pttg.audit.endpoint}") String auditEndpoint,
-                       @Value("${audit.history.endpoint}") String auditHistoryEndpoint,
-                       @Value("${audit.archive.endpoint}") String auditArchiveEndpoint,
-                       @Value("${audit.archive.history.pagesize}") int historyPagesize,
-                       ObjectMapper mapper) {
+                @Value("${audit.history.endpoint}") String auditHistoryEndpoint,
+                @Value("${audit.history.correlationids.endpoint}") String correlationIdsEndpoint,
+                @Value("${audit.history.bycorrelationid.endpoint}") String historyByCorrelationIdEndpoint,
+                @Value("${audit.archive.endpoint}") String auditArchiveEndpoint,
+                @Value("${audit.archive.history.pagesize}") int historyPagesize,
+                ObjectMapper mapper) {
```

Comment:
Is it time for a separate class to hold these properties?  10 constructor args should be breaking checkstyle.

Reply:
Yes it probably is so I will do that now. Interestingly it seems we don't have checkstyle or similar in this project.